portfolio/src/main/java/com/google/sps/servlets/Comment.java

```diff
@@ -0,0 +1,84 @@
+package com.google.sps.servlets;
+
+/**
+ * Encapsulated the possible values for reasons a user visited a site.
+ * Update this enum when adding or removing allowable Visit Type values to the
+ * Comments form.
+ *
+ * NONE represents a null response (i.e. user did not answer the question)
+ * All other responses correspond with one radio button on the form
+ */
+enum VisitType {
+  NONE, RECRUITING, PROJECT, TUTORING, CHAT
+}
+
+/**
+ * Encapsulates a user-entered comment and their personal information,
+ * including their first and last name, reason for visiting, and email address.
+ *
+ * Setter functions included, but getter functions are not needed
+ * since GSON accesses private members directly.
+ */
+public class Comment {
+  private String email;
+  private String firstName;
+  private String lastName;
+  private VisitType visitReason;
+  private String comment;
+
+  /**
+   * Create new Comment object.
+   * Initialize all string values as empty strings
+   * and set visitReason to NONE.
+   */
+  public Comment() {
```

Comment:
If you plan on initializing the fields only once, prefer getting their values via a constructor and making this class immutable.

Reply:
Agreed - didn't know these existed! Fixed!